tests/_utils/logs.py

```
@@ -0,0 +1,13 @@
+import json
+import os
+
+
+def clear_logs() -> None:
+    """Find and clear the log file."""
+    dir = os.path.dirname(__file__)
+    config = json.loads(open(os.path.join(dir,"test_config.json")).read())
```

Ruff

Use a context manager for opening files

(SIM115)